roles/wazuh/ansible-wazuh-manager/tasks/main.yml

```diff
@@ -200,6 +200,16 @@
   tags:
     - config
 
+- name: Retrieving external API Credentials
```

Comment:
I think we should use default vars and avoid importing vars until we reorganize all the default variables.

Please check my next comment, thanks

Reply:
This has been added for pulling vaulted credentials